ephios/plugins/questionnaires/templates/questionnaires/disposition_participation_answers.html

```diff
@@ -0,0 +1,15 @@
+{% load questionnaires_extras %}
+
+<h4>Questionnaire Answers</h4>
```

Comment:
i think a smaller heading would do

Reply:
Oh, I think the size of the other labels (individual start time etc) in the card has changed with the rebase. In the new layout, I agree that this heading is to large.

Changed in ca8906a972719757cce4f2b3901bb6c3f16f247b